src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -665,6 +682,9 @@ export default class PhotoField extends React.Component {
                 aria-valuemin={this.state.minRatio}
                 aria-valuemax={this.state.maxRatio}
                 aria-valuenow={this.state.zoomValue}
+                onMouseDown={this.onZoomMouseDown}
+                onMouseUp={this.onZoomMouseUp}
+                onMouseMove={this.onZoomMouseMove}
                 onInput={this.onZoomSliderChange}/>
```

Comment:
If this isn't firing in IE, does that mean that using the arrow keys to adjust the slider is also not working?

Reply:
Good catch, will replace onInput with onChange, which will capture keyboard input.